tests/models/lvae/test_noise_model.py

```diff
@@ -28,6 +28,12 @@ def create_dummy_noise_model(
     np.savez(tmp_path / "dummy_noise_model.npz", **nm_dict)
 
 
+def test_factory_no_noise_model():
```

Comment:
@CatEek is there a particular reason why we want this behavior? In my opinion this is more logical:
- if `MultiChannelNMConfig == None`, then the noise model is `None` (see my comments in the previous review)
- if `MultiChannelNMConfig.noise_models == []`, then we simply get an "empty" noise model.

Wouldn't this be more clean and logic?